mino/minoch/rpc.go

```diff
@@ -207,9 +211,19 @@ func (c RPC) Stream(ctx context.Context, memship mino.Players) (mino.Sender, min
 			case env := <-in:
 				for _, to := range env.to {
 					if to.(address).orchestrator {
-						orchRecv.out <- env
+						select {
+						case orchRecv.out <- env:
+						default:
+							fmt.Println("FULL!! (orch)")
+							orchRecv.out <- env
+						}
 					} else {
-						outs[to.String()].out <- env
+						select {
+						case outs[to.String()].out <- env:
+						default:
+							fmt.Println("FULL!!")
```

Comment:
it's not possible to use cryChan here, as it would require to change the struct of a receiver, which then breaks the receive function that selects on different channels...